vllm/model_executor/parallel_utils/communication_op.py

```diff
@@ -86,47 +86,55 @@ def tensor_model_parallel_gather(input_: torch.Tensor,
     return output_tensor
 
 
-def broadcast(input_: torch.Tensor, src: int = 0):
+def broadcast(input_: torch.Tensor, src: int = 0, group=None):
```

Comment:
Please add type annotation, and ditto below
```suggestion
def broadcast(input_: torch.Tensor, src: int = 0, group: Optional[ProcessGroup] = None):
```

Reply:
Thanks for reviewing! I committed the new changes as you suggested.